pegen/testutil.py

```diff
@@ -70,7 +70,7 @@ def generate_parser_c_extension(rules, path):
 
 
 def print_memstats() -> bool:
-    MiB: Final = 2 ** 20
```

Comment:
This is needed because master is currently failing due to this type being unavailable.

Reply:
Oh actually the right fix is to add `from __future__ import annotations` to the top of the file.

Reply:
Alas, that's not true, mypy will still complain. The true fix is to add
```
from typing_extensions import Final
```